techniques/failures/F113.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html><html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml"><head><title>Failure of Success Criterion 1.2.5 due to not using available pauses in dialogue to provide audio descriptions of important visual content</title><link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"/></head><body>
+   <h1>Failure of Success Criterion 1.2.5 due to not using available pauses in dialogue to provide audio descriptions of important visual content</h1>
+   <section class="meta"><p class="id">ID: F113</p><p class="technology">Technology: failures</p><p class="type">Type: Failure</p></section>
+   <section id="applicability"><h2>When to Use</h2>
+      <p> Applies to all synchronized media. </p>
+   </section>
+   <section id="description"><h2>Description</h2>
+      <p>This describes a failure condition for all techniques involving audio descriptions. Audio descriptions can either be provided as part of the soundtrack's original narration or be added to audio during pauses in existing dialogue. If important actions, characters, scene changes, and on-screen text are only conveyed visually, appropriate pauses in dialogue need to be used to provide this information as audio descriptions.</p>
+        <p class="note">Not all pauses are usable for audio descriptions. If the pauses in dialogue are too short (for instance, less than 2 seconds), or do not occur in proximity to the visual content that needs to be described, they may not be appropriate for audio descriptions.</p>
+        <p>In situations where important visual information is being conveyed at the same time as non-spoken audio (such as music and sound effects), the technique of "audio ducking" can be used. This involves dropping the overall sound level so that it is easier to distinguish the narration that is added during pauses in dialogue.</p>
+        <p>This technique can work well with background music and sounds, but audio ducking has the potential to mask important audio information. Often such audio information can convey much of the sense of what is visually happening. There may be some pauses in dialogue where non-spoken audio information is so important that the addition of audio descriptions is unnecessary or inappropriate.</p>
+
+</p>
+   </section>
+    
+   <section id="tests"><h2>Tests</h2>
+      <section class="procedure"><h3>Procedure</h3>
+         <p>For each occurrence of synchronized time-based media:</p>
+         <ol>
+            <li> Check that all important visual information has been conveyed in the narration.</li>
```

Comment:
Check that all visual information necessary to understand the content is conveyed by audio in the synchronized media.

Reply:
I added in a variation (below) to try to address this.